optimizely/event_builder.py

```diff
@@ -139,58 +151,57 @@ class EventParams(object):
     CUSTOM = 'custom'
     ANONYMIZE_IP = 'anonymize_ip'
 
-  def _add_attributes(self, attributes):
-    """ Add attribute(s) information to the event.
+  def _get_attributes(self, attributes):
+    """ Get attribute(s) information.
 
     Args:
       attributes: Dict representing user attributes and values which need to be recorded.
+
+    Returns:
+      list Valid attributes for the user (if any). Otherwise, empty.
```

Comment:
This commenting style is not common in Python. Please change this to 

`List consisting of valid attributes for the user. Empty otherwise.`